drawing_1.css

```diff
@@ -0,0 +1,26 @@
+.container{
+	width: 250px;
+	height: 100px;	
+}
+.red{
+	background-color: #8B0000;
+	height: 100px;
+	width: 125px;
+}
+.green{
+	background-color: #008000;
+	height: 100px;
+	width: 125px;
+	float: right;
```

Comment:
Give the `div` you want on the left `float: left;` and the `div` you want on the right `float:right;`. This makes it clear for other programmers what it was that you were intending (important).